kitchen-terraform.gemspec

```diff
@@ -30,22 +41,25 @@ require "rubygems"
   specification.add_development_dependency "middleman-livereload", "~> 3.4"
   specification.add_development_dependency "middleman-syntax", "~> 3.0"
   specification.add_development_dependency "middleman", "~> 4.2"
-  specification.add_development_dependency "mini_racer", "~> 0.3.0"
-  specification.add_development_dependency "pry-coolline", "~> 0.2"
-  specification.add_development_dependency "pry", "~> 0.10"
-  specification.add_development_dependency "reek", "~> 5.5"
   specification.add_development_dependency "rspec", "~> 3.4"
   specification.add_development_dependency "rufo", "~> 0.7"
   specification.add_development_dependency "simplecov", "~> 0.16.1"
   specification.add_development_dependency "travis", "~> 1.8"
+  specification.add_development_dependency "tty-which", "~> 0.4.0"
```

Comment:
The tty-which gem is a runtime dependency used by the client configuration attribute code; it should remain under runtime.

Reply:
As the development dependencies are used for building the gem, there is no real distinction between development and runtime dependencies. I thought this one was only used for development code, but I can see that is wrong. I'll correct and push.